internal/api/registry/uploads.go

```diff
@@ -417,8 +418,10 @@ func (a *API) handleFinishBlobUpload(w http.ResponseWriter, r *http.Request) {
 		contentLength, err := strconv.ParseUint(contentLengthStr, 10, 64)
 		if err != nil {
 			//COVERAGE: unreachable in unit tests because net/http validates Content-Length header format before sending
-			keppel.ErrSizeInvalid.With("malformed Content-Length: " + err.Error())
-			return
+			err = keppel.ErrSizeInvalid.With("malformed Content-Length: " + err.Error())
+			if respondWithError(w, r, err) {
+				return
+			}
```

Comment:
Nice catch from the lint. But the common pattern to report errors of concrete types is
```suggestion
			keppel.ErrSizeInvalid.With("malformed Content-Length: " + err.Error()).WriteAsRegistryV2ResponseTo(w, r)
			return
```